src/news/components/NewsFeedCard.js

```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from 'react';
+import context from '../context/context';
+
+function NewsFeed() {
+  const { data } = useContext(context);
+
+  return (
+    <div>
+      {data.map((articles) =>
+        <div>
+          <h3>{articles.title}</h3>
+          <img src={articles.urlToImage} alt={articles.title} />
+          <p>{articles.description}</p>
+          <a target="blank" href={articles.url}>Saiba Mais</a>
+          <h5>{articles.publishedAt}</h5>
```

Comment:
Esse `articles` não está representado bem. Aqui você tem somente um artigo, então pra ficar melhor representado, coloca como `article`.

```suggestion
      {data.map((article) =>
        <div>
          <h3>{article.title}</h3>
          <img src={article.urlToImage} alt={article.title} />
          <p>{article.description}</p>
          <a target="blank" href={article.url}>Saiba Mais</a>
          <h5>{article.publishedAt}</h5>
```